opt/scopeserver/dataserver/utils/proto.py

```diff
@@ -0,0 +1,54 @@
+"""
+Functions for converting various objects to proto formatted objects
+"""
+
+import json
+import hashlib
+
+from typing import Dict, Any
+
+from scopeserver.dataserver.modules.gserver import s_pb2
+
+
+def protoize_cell_type_annotation(md, secret: str):
```

Comment:
What is `md`, can you make it more descriptive and type annotate it please? Also function docstring (without types)